lib/modules/keyboardNav.js

```diff
@@ -684,6 +684,12 @@ addModule('keyboardNav', {
 		var thisURL = link.getAttribute('href'),
 			button = (this.options.followLinkNewTabFocus.value) ? 0 : 1;
 
+		// we do this because chrome.tabs.create opens URLs consisting solely of fragments (#foo)
+		// relative to the background page instead of the current location
+		if (thisURL.substring(0, 1) === '#') {
+			thisURL = location.href + thisURL;
```

Comment:
This will break if the href already contains a hash. How about `location.pathname + location.search + thisURL` instead?